File: Java Confidence Course/Exercise-solution/Vessel.java
/**
 * @author Jiayee
 *
 */
public class Vessel {

    private static final String MESSAGE_INIT = "Initialising!";
    private double displacement;

    Vessel(double displacement) {
        this.displacement = displacement;
    }

    /**
     * @param args
     */
    public static void main(String[] args) {
        Vessel vessel = new Vessel(2017);
        System.out.println(vessel.displacement);
    }

}

File: Java Confidence Course/Examples/vessel.js
function Vessel(displacement) {
	this.displacement = displacement; // public
	Vessel.count += 1;
}

Vessel.count = 0;

var v1 = new Vessel(2017);
var v2 = new Vessel(2018);
console.log(v1.displacement);
console.log(Vessel.count);